custom_components/tahoma/config_flow.py

```diff
@@ -61,3 +70,37 @@ async def async_step_import(self, import_config: dict):
                 return await self.async_validate_input(import_config)
             except Exception as exception:  # pylint: disable=broad-except
                 _LOGGER.exception(exception)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle a option flow for TaHoma."""
+
+    def __init__(self, config_entry):
+        """Initialize options flow."""
+        self.config_entry = config_entry
+        self.options = dict(config_entry.options)
+
+        if self.options.get(CONF_UPDATE_INTERVAL) is None:
+            self.options[CONF_UPDATE_INTERVAL] = DEFAULT_UPDATE_INTERVAL
+
+    async def async_step_init(self, user_input=None):
+        """Manage the Somfy TaHoma options."""
+        return await self.async_step_update_interval()
+
+    async def async_step_update_interval(self, user_input=None):
+        """Manage the options regarding interval updates."""
+        if user_input is not None:
+            self.options[CONF_UPDATE_INTERVAL] = user_input[CONF_UPDATE_INTERVAL]
+            return self.async_create_entry(title="", data=self.options)
```

Comment:
You probably take this from dev documentation. Do you know if the empty title is actually store in the config entries file?

Reply:
Yes, but all examples I have seen (like Hue) are using an empty title. It looks like it is not used.